posydon/popsyn/synthetic_population.py

```diff
@@ -392,7 +410,24 @@ def get_dco_merger_efficiency(self):
             efficiencies.append(eff)
             print(f'DCO merger efficiency at Z={met:1.2E}: {eff:1.2E} Msun^-1')
         self.met_merger_efficiency = np.array(self.met_merger_efficiency)
-        self.merger_efficiency = np.array(efficiencies)
+        self.merger_efficiency = {}
+        self.merger_efficiency['total'] = np.array(efficiencies)
```

Comment:
The two lines of creating a dictionary and adding an entry there can be combined in one statement/command:
`self.merger_efficiency = {'total': np.array(efficiencies)}`